.github/workflows/docker.yml

```
@@ -4,52 +4,34 @@ on:
     push:
 
 jobs:
-    docker-compose:
-        name: Excute Tests on Docker Compose
+    static-analyze:
+        name: Execute Static Analyze on Docker Compose
         runs-on: ubuntu-latest
         env:
             COMPOSE_FILE: docker-compose.test.yml
 
         steps:
-            -   name: Checkout code
+            -
+                name: Checkout Code
                 uses: actions/checkout@v2
 
-            -   name: Run PHPUnit, PHPSpec and Behat tests
-                run: docker-compose run ci
+            -
+                name: Run PHPUnit, PHPSpec and Psalm
+                # docker-compose -f docker-compose.test.yml --profile static-analyze run static
+                run: docker-compose --profile static-analyze run static
```

Shouldn't it be like the one below?
```suggestion
                run: docker-compose -f docker-compose.test.yml --profile static-analyze run static
```

---

I use the `COMPOSE_FILE: docker-compose.test.yml` env to "sugar" the code :) The `COMPOSE_FILE` basically points to which file is "default" to the docker compose